go/stellar/stellargregor/stellargregor.go

```diff
@@ -56,3 +60,17 @@ func (h *Handler) autoClaim(mctx libkb.MetaContext, cli gregor1.IncomingInterfac
 	mctx.G().GetStellar().KickAutoClaimRunner(mctx, item.Metadata().MsgID())
 	return nil
 }
+
+func (h *Handler) paymentStatus(mctx libkb.MetaContext, cli gregor1.IncomingInterface, category string, item gregor.Item) error {
```

Comment:
at least initially while the format may still be in flux, it might be good to dismiss these even if they fail to unmarshal

Reply:
done, thanks!